ts/src/pro/binance.ts

```diff
@@ -2060,24 +2060,22 @@ export default class binance extends binanceRest {
          * @returns {object[]} a list of [order structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
          */
         await this.loadMarkets ();
-        let messageHash = 'orders';
-        let market = undefined;
-        if (symbol !== undefined) {
-            market = this.market (symbol);
-            symbol = market['symbol'];
-            messageHash += ':' + symbol;
-            params = this.extend (params, { 'symbol': symbol }); // needed inside authenticate for isolated margin
-        }
-        await this.authenticate (params);
-        let type = undefined;
-        [ type, params ] = this.handleMarketTypeAndParams ('watchOrders', market, params);
+        const defaultType = this.safeString2 (this.options, 'watchOrders', 'defaultType', 'spot');
```

Comment:
@sc0Vu why using this instead of `handleMarketType...`?

Reply:
When use this `handleMarketTypeAndParams`, the type would be `spot` instead of `margin`.